### ISSUE TITLE
Support linux 4.5.2

### ISSUE THREAD

**Issue by matt-42** [archzfs/archzfs#56]
Archlinux has evolved to linux 4.5.2 (https://www.archlinux.org/packages/core/i686/linux/)

Is there any workaround to get the current zfs working with the upgraded archlinux tree?
If not could you update the zfs packages accordingly ?

Here are some errors we get when installing archzfs/zfs-linux:
==> No kernel 4.5.1-1-ARCH modules. You must install them to use DKMS!
depmod: WARNING: could not open /lib/modules/4.5.1-1-ARCH/modules.builtin: No such file or directory

Thanks,
Matthieu

**Comment by seiichiro0185** [comment]
I also ran into this error. 
With the old repository (demz-repo-*) there was a hard dependency to the kernel version which the modules where built for. This meant that kernel-updates were refused, as long as there were no matching zfs-packages available 
With the new repo, this seems to have gone, allowing the kernel update to install even if the modules are not working wih it.
Could this dependency be reintroduced for the packages, because a state like this will lead to a unbootable system if root is on ZFS?

**Comment by demizer** [comment]
Thanks for the report. This is fixed now in the latest updates being pushed soon (next 10 minutes). I returned to the old hard set linux package version.

To understand why this was changed, see https://github.com/archzfs/archzfs/issues/28#issuecomment-217774637

TLDR; This is all my mistake!

**Comment by lnicola** [comment]
Did you mean to update the packages on your repo?

**Comment by lnicola** [comment]
Nevermind, they're ready :).

**Comment by demizer** [comment]
Yep, forgot to push the updated archzfs repo. Not sure why my alarms failed. :(
